<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="APERGHN1UG" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="12 Feb 2021 01:04:55 GMT" time="0.092" errors="0">
  <testcase name="tc222" time="0.016" classname="org.practice.Manualfailrun">
    <failure type="java.lang.AssertionError" message="test case failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: test case failed expected [true] but found [false]
at org.practice.Manualfailrun.tc222(Manualfailrun.java:17)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc222 -->
  <testcase name="tc444" time="0.0" classname="org.practice.Manualfailrun">
    <ignored/>
  </testcase> <!-- tc444 -->
  <testcase name="tc111" time="0.0" classname="org.practice.Manualfailrun">
    <ignored/>
  </testcase> <!-- tc111 -->
  <testcase name="tc333" time="0.0" classname="org.practice.Manualfailrun">
    <ignored/>
  </testcase> <!-- tc333 -->
</testsuite> <!-- Default test(failed) -->
